lern-data-products/src/main/scala/org/sunbird/core/exhaust/OnDemandExhaustJob.scala

```diff
@@ -173,6 +173,8 @@ trait OnDemandExhaustJob {
       // $COVERAGE-OFF$ Disabling scoverage
       case "s3" =>
         CommonUtil.getS3File(storageConfig.container, "")
+      case "oci" =>
+        CommonUtil.getS3File(storageConfig.container, "");
```

Comment:
Code is same for s3 and oci. Please use the pipe in case to match multiple values. 
Sample code:
`code match {
  case "1" | "2" => // whatever
  case "3" =>
}`

Reply:
made changes as suggested